src/main/java/org/quantumbadger/redreader/activities/MainActivity.java

```diff
@@ -247,10 +260,7 @@ public void onSelected(final @MainMenuFragment.MainMenuAction int type, final St
 			case MainMenuFragment.MENU_MENU_ACTION_CUSTOM: {
 
 				final AlertDialog.Builder alertBuilder = new AlertDialog.Builder(this);
-				final LinearLayout layout = (LinearLayout) getLayoutInflater().inflate(R.layout.dialog_editbox, null);
-				final EditText editText = (EditText)layout.findViewById(R.id.dialog_editbox_edittext);
-
-				editText.requestFocus();
```

Comment:
I set `<requestFocus />` via xml instead as every time this `EditText` was referenced, `requestFocus()` was being called, so doing it via xml is more explicit.